benchmarks/benchmark_throughput.py

```diff
@@ -84,6 +84,7 @@ def run_vllm(
     gpu_memory_utilization: float = 0.9,
     download_dir: Optional[str] = None,
     load_format: str = EngineArgs.load_format,
+    disable_output_proc_callback: bool = False,
```

Comment:
Since we implemented the overlap of `process_model_outputs` and GPU forward pass using a callback function and its not really "asynchronous", I think `disable_output_proc_callback` is a better choice.

Reply:
Changed to `disable_async_output_proc`